README.md

```diff
@@ -2328,6 +2328,21 @@ class API < Grape::API
 end
 ```
 
+For using Rack::Sendfile middleware, specify `to_path` method in your
```

Comment:
But this is not true, no? The code change makes it that I don't have to worry about file streamers or any other thing, I can just plop `Rack::Sendfile` in there and it will "just work", right? Maybe just say that this is compatible with Rack::Sendfile and add a note on what that does?